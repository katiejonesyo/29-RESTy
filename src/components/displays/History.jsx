import React from 'react';
import propTypes from 'prop-types';

const History = ({ url, method }) => (
    <section>
        <div>
            <p>{url}</p>
            <p>{method}</p>
        </div>
    </section>
);

History.protoType = {
    url: propTypes.string.isRequired,
    method: propTypes.string.isRequired
};

export default History;


