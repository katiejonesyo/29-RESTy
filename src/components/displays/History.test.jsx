
import React from 'react';
import { render, cleanup } from '@testing-library/react';
import History from './History'

describe('History component', () => {
  afterAll(() => cleanup());

  it('renders History', () => {
    const { asFragment } = render(<History
      url="www.url.com"
      method="GET"
    />);
    expect(asFragment()).toMatchSnapshot();
  })
});


