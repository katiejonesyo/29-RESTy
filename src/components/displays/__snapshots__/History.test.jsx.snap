// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History component renders History 1`] = `
<DocumentFragment>
  <section>
    <div>
      <p>
        GET
      </p>
      <p>
        www.url.com
      </p>
    </div>
  </section>
</DocumentFragment>
`;


